pkg/kev/init_test.go

```diff
@@ -216,11 +217,31 @@ services:
       kev.workload.liveness-probe-command: '["CMD", "echo", "Define healthcheck command for service db"]'
       kev.workload.liveness-probe-type: exec
       kev.workload.replicas: "1"
+    x-k8s:
+      enabled: true
+      workload:
+        type: Deployment
+        replicas: 1
+        livenessProbe:
+          type: exec
```

Comment:
I appreciate it may not be in scope of this PR, but something we should consider.

Instead of specifying the probe type we could just use it as a key with nested configuration specific to selected probe type, see example in https://github.com/appvia/kev/issues/443.

```yaml
livenessProbe:
  exec:
    command: "your command"
...

readinessProbe:
  http:
    port: 8080
    path: /
...

readinessProbe:
  tcp:
    port: 8080
...
```

The only challenge with the above is a `none` probe type... but perhaps we could work on assumption that if `none` type was intended, then the probe shouldn't be present in the configuration at all?

Reply:
I'd rather table this for now and discuss this as a separate issue. This causes a bit of problems because many of them can be filled and so we end up with more errors and making it really hard for someone to switch between types because they would need to remove all the alternatives or risk them being used on the k8s config. Not sure if that produces an error or not. Either case I think this is better as a ticket to be discussed, what do you think?